cpp/daal/src/algorithms/dbscan/dbscan_distributed_input_types.cpp

```diff
@@ -526,19 +526,17 @@ services::Status DistributedInput<step5Local>::check(const daal::algorithms::Par
     const size_t nDataBlocks = dcPartialData->size();
     DAAL_CHECK_EX(nDataBlocks > 0, ErrorIncorrectNumberOfInputNumericTables, ArgumentName, partialDataStr());
 
-    size_t nFeatures = 0;
-    for (size_t i = 0; i < nDataBlocks; i++)
+    size_t nFeatures = NumericTable::cast((*dcPartialData)[0])->getNumberOfColumns();
+    if (NumericTable::cast((*dcPartialData)[0])->getNumberOfRows() != 0)
```

Comment:
it is reasonable to put this check inside loop and just skip tables with 0 rows.